### ISSUE TITLE
yAxis Labels not static while panning

### ISSUE THREAD

**Issue by Kalleno** [chartjs/chartjs-plugin-zoom#275]
Hey guys!

I have an weird issue while panning after upgrading chartjs and the zoom plugin to the newest versions (2.8.0 and 0.7.4). Here an quick GIF of my problem. The yAxis is not static, its hiding from time to time while panning the chart.

![pan issue](https://user-images.githubusercontent.com/6626403/65859477-95275e80-e368-11e9-915a-2a39d4a8b997.gif)

Is there something I totally miss? I'm the only one with that weird behavior? Any help is appreciated!

Here my chart config. Some of these options get changed based on the data in the chart dynamically. 
 
```javascript
options: {
    pan: {
        enabled: true,
        mode: 'x',
        speed: 20,
        threshold: 1,
        rangeMin: {
            x: moment().startOf('year').subtract('1', 'year').unix() * 1000,
            y: null
        },
        rangeMax: {
            x: moment().endOf('year').add('1', 'month').unix() * 1000,
            y: null
        }
    },
    zoom: {
        enabled: false
    },
    title: {
        display: false
    },
    legend: {
        display: false,
        labels: {
            boxWidth: 20
        }
    },
    maintainAspectRatio: false,
    hover: {
        mode: null
    },
    responsive: true,
    tooltips: {
        mode: 'x',
        callbacks: {
            title: function(),
            label: function()
        },
        enabled: false,
        custom: function()
    },
    scales: {
        xAxes: [{
            type: 'time',
            distribution: 'linear',
            display: true,
            stacked: true,
            scaleLabel: {
                display: false
            },
            time: {
                unitStepSize: 1,
                unit: 'month',
                tooltipFormat: 'll HH:mm',
                min: moment('2018-01-01'),
                max: moment('2018-04-01'),
            },
            ticks: {
                bounds: 'ticks',
                source: 'auto',
                maxRotation: 0,
                minRotation: 0
            },
            gridLines: {
                drawBorder: false,
                color: 'rgba(0,0,0,0)'
            },
            barThickness: 8
        }],
        yAxes: [{
            type: 'linear',
            id: 'y-axis-0',
            scaleLabel: {
                display: false
            },
            ticks: {
                fontFamily: 'Raleway',
                fontSize: 10,
                beginAtZero: true,
                stepValue: 5,
                min: 0,
                max: 500
            },
            afterBuildTicks: function(chart) {
                chart.ticks.splice(0, 1);
            },
            gridLines: {
                drawBorder: true,
            }
        }]
    }
}

**Comment by benmccann** [comment]
That is weird. I haven't seen that before

Can you share a codepen, plunker, or similar containing this example?

If you figure out the issue and want to send a PR, I'd help review it

**Comment by Kalleno** [comment]
Here's a codepen: https://codepen.io/t_simon/pen/YzKojPY

This is causing the error: 
`
config.options.scales.yAxes[0].afterFit = function(scaleInstance) {
	scaleInstance.width = 0; 
};`

Sorry for the bugreport! In the end, it was an fault on our side. I dont know, why the _afterFit_ function was in there. It is an project with a dev team. I removed it, now it works great!

**TL;DR:** Dont change the char tinstance's width to zero after initialization.
